src/components/HomeSection.test.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import HomeSection from './HomeSection';
+
+describe('HomeSection', () => {
+  test('checks if the title is in the document', () => {
+    render(
+      <HomeSection
+        title="Topology"
+        description="Math course"
+        buttons={[]}
+        pic="manifold"
+      />
+    );
+    screen.debug();
+    expect(screen.getByText('Topology')).toBeInTheDocument();
+  });
+
+  test('checks if the description is in the document', () => {
+    render(
+      <HomeSection
+        title="Topology"
+        description="Math course"
+        buttons={[]}
+        pic="manifold"
+      />
+    );
+    expect(screen.getByText('Math course')).toBeInTheDocument();
+  });
+
+  test('checks if it renders the given number of buttons', () => {
```

Comment:
A few more (optional) test case ideas:
- Renders given pic
- Renders given tab name (that `tab` property should maybe be called `tabName`)

The defaults can be part of that "default render" test case.

These are possible test cases too, but these are kind of obscure and might be a bit harder to test so you can skip these if you like:
- Renders in given direction
- Renders in given alignItems